stock_move_name_from_sale_line/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+This module set stock move name from sale order line when is confirmed.
```

Comment:
Minor remarks : Could you mention what is the (default) name value, when this module is not installed ?